File: Iniciante/1017 - Gasto de Combustível/Python/1017.py
def main():
    hours_spent_traveling = int(input())
    average_speed = int(input())
    fuel_required = calculate_fuel_required(hours_spent_traveling, average_speed)
    print(f"{fuel_required:.3f}")


def calculate_fuel_required(hours_spent_traveling, average_speed):
    KILOMETERS_PER_LITER = 12.0
    return (hours_spent_traveling * average_speed) / KILOMETERS_PER_LITER


if __name__ == "__main__":
    main()

File: Iniciante/1017 - Gasto de Combustível/JavaScript/1017.js
function main() {
    const input = require("fs").readFileSync("/dev/stdin", "utf8");
    const lines = input.split("\n");
    const [hoursSpentTraveling, averageSpeed] = lines.map(Number);
    const fuelRequired = calculateFuelRequired(hoursSpentTraveling, averageSpeed);
    console.log(`${fuelRequired.toFixed(3)}`);
}

function calculateFuelRequired(hoursSpentTraveling, averageSpeed) {
    const  KILOMETERS_PER_LITER = 12.0;
    return (hoursSpentTraveling * averageSpeed) / KILOMETERS_PER_LITER;
}

main();